packages/internal/bridge/bridge-sample-app/.example.env

```diff
@@ -59,4 +59,9 @@ FLOW_RATE_TOKEN_DECIMALS=18
 # Values are in whole tokens and will be converted to wei using the decimals above
 FLOW_RATE_CAPACITY=15516
 FLOW_RATE_REFILL_RATE=4.31
-FLOW_RATE_LARGE_TRANSFER_THRESHOLD=7758
\ No newline at end of file
+FLOW_RATE_LARGE_TRANSFER_THRESHOLD=7758
```

Comment:
The large transfer threshold currently configured for USDC on Testnet is `10008`. Do you want to use that instead?

Reply:
can if we want but these are just examples and the settings should be figured out per token you're wanting to map though.